### ISSUE TITLE
[META] Global header accessibility issue

### ISSUE THREAD

**Issue by obulat** [cc-archive/vocabulary-legacy#728]
## Description
This is a parent issue for the accessibility problems in Global header: 

Related issues are here:

- [x] #717 Hidden global header links should not be focusable
- [x] #718 Button focus indicator is invisible on black background
- [x] #727 ARIA attributes for global header
## Resolution

- [ ] I would be interested in resolving this bug.

**Comment by obulat** [comment]
All the linked issues have been solved!
